_includes/assets-site-head.html

```diff
@@ -1,6 +1,6 @@
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.1.1/normalize.min.css">
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic&subset=latin,latin-ext" type="text/css">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
-<link rel="stylesheet" href="{{ '/css/styles.css?v=' | append: site.github.build_revision | relative_url }}">
+<link rel="stylesheet" href="/dest/css/libraries-main.css">
```

Comment:
Why are we using this link rather than the template-language version? (Also, my local Jekyll complains that it is not found.)